libdnf5-cli/output/packageinfo.cpp

```diff
@@ -52,25 +52,25 @@ void print_package_info(
     scols_table_enable_noheadings(table, 1);
     scols_table_new_column(table, "key", 1, 0);
     scols_table_new_column(table, "value", 1, SCOLS_FL_WRAP);
-    scols_table_set_column_separator(table, " : ");
+    scols_table_set_column_separator(table, _(" : "));
```

Comment:
Is it expected that the translators know what purpose this string serve or should a note, such as https://github.com/rpm-software-management/dnf5/blob/main/dnf5daemon-client/commands/advisory/arguments.hpp#L153-L154 , be added?

Reply:
Thank you!